addons/sourcemod/scripting/AntiBhopCheat_Discord.sp

```diff
@@ -0,0 +1,212 @@
+#include <utilshelper>
+#include <discordWebhookAPI>
+
+#undef REQUIRE_PLUGIN
+#tryinclude <SelectiveBhop>
+#tryinclude <ExtendedDiscord>
+#tryinclude <sourcebanschecker>
+#define REQUIRE_PLUGIN
+
+#define PLUGIN_NAME "AntiBhopCheat Discord"
+#define WEBHOOK_URL_MAX_SIZE			1000
+#define WEBHOOK_THREAD_NAME_MAX_SIZE	100
+
+ConVar g_cvCountBots, g_cvWebhook, g_cvWebhookRetry, g_cvChannelType;
+ConVar g_cvThreadName, g_cvThreadID, g_cvAvatar;
+
+char g_sMap[PLATFORM_MAX_PATH];
+bool g_Plugin_SourceBans = false;
+bool g_Plugin_ExtDiscord = false;
+
+public Plugin myinfo =
+{
+	name			= PLUGIN_NAME,
```

Comment:
define not needed, if you want to access this information, you can use:
- https://sourcemod.dev/#/sourcemod/function.GetPluginInfo
- https://sourcemod.dev/#/core/enumeration.PluginInfo